core/src/main/java/io/questdb/cairo/mv/MatViewDefinition.java

```diff
@@ -48,13 +48,17 @@
 import static io.questdb.std.datetime.microtime.Timestamps.MINUTE_MICROS;
 
 public class MatViewDefinition implements Mutable {
-    // For now, incremental refresh is the only supported refresh type.
+    public static final int INCREMENTAL_INTERVAL_REFRESH_TYPE = 1;
     public static final int INCREMENTAL_REFRESH_TYPE = 0;
```

Comment:
All refreshes are incremental, aren't they? The only difference is that some of them are timer-triggered, some are manually triggered, and some are automatically triggered.

Reply:
It's now called "incremental_timer" in `materialized_views()` output.